### ISSUE TITLE
bug - custom indexes not properly used when creating Virtual Tenants from the user interfaces for splk-dsm/dhm/mhm

### ISSUE THREAD

**Issue by sebwurl** [trackme-limited/trackme-report-issues#30]
Hi,
I'm creating a new tenant with custom indexes in v.2.0.9.
The filter of the dropdown is not working (I can't access the search box of the dropdown). I can select indexes manually, but data is written to the default indexes afterwards. For example clicking on "Summary Index" in the tenant overview, shows data from index trackme_summary.

This is how it looks like during tenant creation:
![image](https://user-images.githubusercontent.com/85566586/222132452-c6f353c3-692c-4130-bd2b-a5ba13c83639.png)

![image](https://user-images.githubusercontent.com/85566586/222133459-632ae8c1-36e7-4b68-967c-e1ca67d99cdb.png)

**Comment by guilhemmarchand** [comment]
Will check carefully ASAP @sebwurl 

Apart from the search in the dropdown, you are saying the data is not going where it should, does that happen only for the summary events?

**Comment by guilhemmarchand** [comment]
Ok I think I got it

TrackMe is leveraging an endpoint for that:

```
| trackme url="/services/trackme/v2/vtenants/tenant_idx_settings" mode="post" body="{'tenant_id': 'mytenant'}"
```

There must be a regression therefore it returns the normal default indexes - checking for it

**Comment by guilhemmarchand** [comment]
You probably see this in the REST logs:
 
 ```
 2023-03-01 15:56:18,502 ERROR trackme_rest_handler_vtenants.py post_tenant_idx_settings 403 Failed to load tenant_idx_settings="{\\\"trackme_summary_idx\\\": \\\"trackme_custom_summary\\\", \\\"trackme_audit_idx\\\": \\\"trackme_custom_audit\\\", \\\"trackme_notable_idx\\\": \\\"trackme_custom_notable\\\", \\\"trackme_metric_idx\\\": \\\"trackme_custom_metrics\\\"}" with exception="Expecting property name enclosed in double quotes: line 1 column 2 (char 1)"
 ```

**Comment by guilhemmarchand** [comment]
Checking it - straightforward fix
Once address we will publish urgently the fix release

**Comment by guilhemmarchand** [comment]
Will be fixed shortly, UI related issues when setting up the REST call
I will print t-shirts for you @sebwurl  ;-)

**Comment by guilhemmarchand** [comment]
All fixed in version 2.0.10

Thanks for reporting @sebwurl !
